### ISSUE TITLE
Adjust our standard bottom sheet to the new UI guidance

### ISSUE THREAD

**Issue by sonora** [osmandapp/OsmAnd#16331]
We need to adjust our standard bottom sheets e.g. for general settings `general_profile_settings.xml` to the new look, namely allowing to show the icons everywhere.
- [ ] For that need some guidance if this is the correct default scheme with icons on the left, radio buttons on the right, etc.

<img width="225" alt="Screenshot 2023-01-26 171301" src="https://user-images.githubusercontent.com/845735/214888347-646fdaa3-ddae-4eea-8924-d7d496bb5e96.png">

**Comment by vshcherb** [comment]
Merged back to https://github.com/osmandapp/OsmAnd/issues/16157, obviously there are 2 types of dialog with icons on the left and radio on the right and no icon / radio on the left. Which should be take as  standard or redesigned probably a technical question but not important

**Comment by sonora** [comment]
Sure. And yes, my understanding is that _no icon / radio on the left_ is supposed to be phased out and replaced by _icon on left / radio on right_.

This issue simply meant to intiatiate and track that across all (at least profile general) settings rather than leaving this task unaligned as side note in several case-by-case issues like #15836, #16157, #15724 (latter one already closed without corresponding adjustment).

**Comment by sonora** [comment]
I'm confused now. That contradicts with the guidance of screenshots posted here https://github.com/osmandapp/OsmAnd/issues/15836#issuecomment-1385619073, which explicitly requested to replace the former custom dialog (depicted in the screenshot above that post). And also with this guidance https://github.com/osmandapp/OsmAnd/issues/16157#issuecomment-1401682405?

As is publicly visible, I invest quite a significant number of pro bono hours to support the dev team achieving our goals in times of short resources. It would really help to have some consistent guidance to reduce the risk of wasting these hours on something which eventually is not the desired implementation.

**Comment by vshcherb** [comment]
It's very confusing and time consuming cause UI is done in several steps. UI Preparation -> Approving -> Implementation, here we jump to implementation phase and then fixing and then we get back to UI Preparation. 

Here I see we created artificial technical task which only confuses and is not required, once @Chumva 
picks up https://github.com/osmandapp/OsmAnd/issues/15836, he will prepare bottom sheet needed for Map Orientation. And I answered there is no goal to have all bottom sheets the same, there will be still bottom sheets with icon and without icons. 

It's probably if you talk to @dmpr0 directly, share ideas and UI concerns though anyway UI needs to be "properly documented" and not addressed by developers.
